packages/template-retail-react-app/app/page-designer/README.md

```diff
@@ -0,0 +1,100 @@
+
+- **/assets** - This folder contains the non-visual components that you dragged and dropped on your **Page Designer** pages. These include things like "Image", "ImageWithText" and and any other Page Designer assets you want to visualize in your PWA-Kit app. If you need to visualize a new component that we haven't provided, you can add it here.
+
+- **/layouts** - Contrary to the above, this folder contains only components that involve layout of other components. We have gotten you off to a good start by giving you various _grid_ and _carousel_ layout components.
+
+## Sample Usage
+
+Below is a sample page that you can use in your _retail react app_ to visualize pages that have created in business manager.
+
+```
+// app/pages/page-viewer/index.jsx
+
+import React from 'react'
+import {Box} from '@chakra-ui/react'
+import {Page, pageType} from '../../page-designer'
+import {ImageTile} from '../../page-designer/assets'
+import {
+    Carousel,
+    MobileGrid1r1c,
+    MobileGrid2r1c,
+    MobileGrid2r2c,
+    MobileGrid2r3c,
+    MobileGrid3r1c,
+    MobileGrid3r2c
+} from '../../page-designer/layouts'
+
+const PageViewer = ({page}) => {
+    // Assign the components to be rendered for any given page designer component type.
+    const components = {
+        'commerce_assets.photoTile': ImageTile,
+        'commerce_assets.imageAndText': ImageWithText,
+        'commerce_layouts.carousel': Carousel,
+        'commerce_layouts.mobileGrid1r1c': MobileGrid1r1c,
+        'commerce_layouts.mobileGrid2r1c': MobileGrid2r1c,
+        'commerce_layouts.mobileGrid2r2c': MobileGrid2r2c,
+        'commerce_layouts.mobileGrid2r3c': MobileGrid2r3c,
+        'commerce_layouts.mobileGrid3r1c': MobileGrid3r1c,
+        'commerce_layouts.mobileGrid3r1c': MobileGrid3r2c
+    }
+
+    return (
+        <Box layerStyle={'page'}>
+            <Page page={page} components={components} />
+        </Box>
+    )
+}
+
+PageViewer.getProps = async ({api, params}) => {
+    const {pageId} = params
+    const page = await api.shopperExperience.getPage({
+        parameters: {pageId}
+    })
+
+    return {
+        page
+    }
+}
+
+PageViewer.displayName = 'PageViewer'
+
+PageViewer.propTypes = {
+    page: pageType.isRequired
+}
+
+export default PageViewer
+```
+
+_**Note:** To ensure that this page is routeable you need to add it to your applications routes file._
+
+```
+// app/routes.jsx
+
+// Create new loadabled page for the `page-viewer`
++ const PageViewer = loadable(() => import('./pages/page-viewer'), {fallback})
+
+
+// Add the new route object to the routes array
++    {
++        path: '/page-viewer/:pageId',
++        component: PageViewer
++    },
+```
```

Comment:
Do we want to include instructions to visualize the page? 
E.g.

1. Run the dev server. 
```
npm run start --prefix packages/template-retail-react-app/
```
2. Navigate to the page-viewer URL with the Page designer page id.
http://localhost:3000/page-viewer/{page-id}
http://localhost:3000/page-viewer/homepage-example

Reply:
I can probably add a note about the route, but I think stating the dev server is covered elsewhere.

Reply:
I add some additional verbage for those that might take the sample code for production code.